site/content/3.13/graphs/graph-visualizer/_index.md

```diff
@@ -0,0 +1,150 @@
+{{< image src="../../images/1a-Create_Node.png" >}}
+
+- **Step-2 : Select Create Node**
+{{< image src="../../images/1b-Create_Node.png" >}}
+
+- **Step-3 : Select the desired Node Type and click on Create**
+{{< image src="../../images/1c-Create_Node.png" >}}
+
+**B.Add New Edges (Relationships):**  
+  Right-click on the canvas and select **"Create Edge"**. Choose the edge collection from the dropdown, provide a name for the edge, and correctly set the `_from` and `_to` fields by selecting source and target nodes. The edge will be created and visualized between the two nodes.
+
+- **Step-1 : Right Click On Graph Area**
+{{< image src="../../images/2a-Create_Edge.png" >}}
+
+- **Step-2 : Select Create Edge**
+{{< image src="../../images/2b-Create_Edge.png" >}}
+
+- **Step-3 : Select the desired Edge from drop down and enter valid details in '_from' and '_to' and click Create**
+{{< image src="../../images/2c-Create_Edge.png" >}}
+
+**C.Delete Nodes or Edges:**  
+  Select a node or edge and right-click to access the **"Delete"** option. This action removes the selected element from both the graph and the database.
+
+- **Select Node and Right click**
+{{< image src="../../images/3a-Delete_Node.png" >}}
+
+- **Select Edge and Right click**
+{{< image src="../../images/3b-Delete_Edge.png" >}}
+
+**D.Properties of Nodes or Edges:**  
+  If You Select a node or edge a Pop-In will appear and display the properties of selected Node or Edge.
+
+- **Select Edge to display the Properties**
+{{< image src="../../images/4a-Properties-of-Edge.png" >}}
+
+- **Select Node to display the Properties**
+{{< image src="../../images/4-Properties-of-Node.png" >}}
+
+## 2.Graph Visualization
+
+The core function of the Graph Visualizer is to provide an intuitive canvas for exploring graph structures.
+
+- **List All Graphs:**  
+  View and select any graph from the list of connected graphs, including **General Graphs**, **Smart Graphs**, **Satellite Graphs**, **Enterprise Graphs**, and **Knowledge Graphs**.
+{{< image src="../../images/1_List All Graphs.png" >}}
+
+- **Select and Load a Graph:**  
+  Upon selecting a graph, its nodes and edges are visualized. The visualization includes a summary of the vertex and edge collections involved, providing a clear view of the data model and relationships.
+{{< image src="../../images/2_Load_Graph.png" >}}
+
+- **Dynamic Graph Rendering:**  
+  In addition to static graphs, you can dynamically render graphs using the **New Query** or **Saved Queries** options. This allows visualizing results from custom AQL queries on demand.
+  - **Dynamic Graph By using New-Query**
+{{< image src="../../images/3a_DynamicGraphby_NewQuery.png" >}}
+  - **Dynamic Graph By using Saved-Query**
+{{< image src="../../images/DynamicGraph_By_Saved_Query.png" >}}
+
+## 3.Search & Filter Data
+
+The top-left section of the Graph Visualizer includes powerful search and query tools for interactive exploration.
+
+##### 🔍 Search
```

Comment:
- Please don't skip headline levels. Pages should have a sound structure.
- I'm not sure about the use of emoji